background.js

```diff
@@ -6,9 +6,14 @@ var count = 0;
 $("#banner").fadeTo(4000, 500).slideUp(500, function(){
   $("#banner").slideUp(500);
 });
+
 $("#developer").fadeTo(4000, 500).slideUp(500, function(){
   $("#developer").slideUp(500);
 });
+
+/**
+ * Retrieves all chrome cookies and displays the amount in the 'cookie-counter' element.
+ */
```

Comment:
Can you please add reference URL to `chrome.cookies.getAll` API?
It'll help readers.